packages/@aws-cdk/aws-ec2-alpha/README.md

```diff
@@ -231,6 +231,63 @@ new Route(this, 'DynamoDBRoute', {
 });
 ```
 
+## VPC Peering Connection
+
+VPC peering connection allows you to connect two VPCs and route traffic between them using private IP addresses. The VpcV2 construct supports creating VPC peering connections through the `VPCPeeringConnection` construct from the `route` module.
+
```

Comment:
You should explain the 3 cases of VPCs peering (same account, same region, Cross regions, cross accounts). Add the steps for how to use CDK to create the 3 cases.

Reply:
done! [b926d86](https://github.com/aws/aws-cdk/pull/31645/commits/b926d864a3746cef4aaeee2aa55e9ed84c434842)